### DISCUSSION TITLE
log4j

### DISCUSSION THREAD

**Discussion by ktpatrick** [nightscape/spark-excel#508] [Q&A] [answered]
I'm trying to understand if spark-exel has been updated to use the patched log4j>=2.15.0? and if so, any changes need to incorporate this update. Thanks!

**Comment by quanghgx** [comment, marked as answer]
For related upstream ticket (and discussion): https://issues.apache.org/jira/browse/SPARK-37630
It seems that, this is not spark-excel specific, and spark-excel by itself not explicitly requires log4j nor provide using log4j configuration with JMSAppender.

However, we might need to put a clear note somewhere to warn spark-excel users of this potential issue.
